uptrain/operators/language/model_grade.py

```diff
@@ -105,7 +105,7 @@ def setup(self, settings: Settings):
     def _make_payload(self, id: t.Any, messages: list[dict]) -> Payload:
         return Payload(
             endpoint="chat.completions",
-            data={"model": "gpt-3.5-turbo", "messages": messages},
+            data={"model": "gpt-3.5-turbo", "messages": messages, "temperature": 0.2},
```

Comment:
temperature needs to be the variable declared earlier?

Reply:
No, for model grading, we can fix the temperature. It is defined for response generation only